db/version_builder_test.cc

```diff
@@ -1655,6 +1654,96 @@ TEST_F(VersionBuilderTest, EstimatedActiveKeys) {
             (kEntriesPerFile - 2 * kDeletionsPerFile) * kNumFiles);
 }
 
+class FilePreloadTest : public DBTestBase {
+ public:
+  FilePreloadTest() : DBTestBase("/file_preload_test", /*env_do_fsync=*/true) {}
+};
+
+TEST_F(FilePreloadTest, PreloadCaching) {
+  // create a DB with 3 files
+  // the day.
```

Comment:
the day?

Reply:
No clue how that got in there ... both instances removed.